pallets/accounts/src/lib.rs

```diff
@@ -496,6 +517,52 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Update 'data'
+		///
+		/// TODO
+		#[pallet::weight(25_000)] // TODO #1 - weight
```

Comment:
why not call this extrinsic `link_account` ?

Reply:
this function looks correct.

But just to confirm, when the Linked External User Account (for e.g the Discord Account) wants to use the Linked External User Account to call the extrinsic `<EXTRINSIC HE WANTS TO CALL>` - 

he will have to call the extrinsic: 

`pallet_proxy::Pallet::proxy(<CLAMOR ACCOUNT ID THAT IS LINKED TO THE ANON PROXY>, <THE ANON_PROXY>, T::ProxyType, <EXTRINSIC HE WANTS TO CALL>)`

is this correct @sinkingsugar?